modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminSelector.tsx

```diff
@@ -4,59 +4,135 @@
  */
 
 import ClayButton from '@clayui/button';
+import {Option, Picker} from '@clayui/core';
 import ClayDropDown from '@clayui/drop-down';
 import ClayIcon from '@clayui/icon';
-import ClayLabel from '@clayui/label';
 import ClayLayout from '@clayui/layout';
-import React, {useEffect, useMemo, useState} from 'react';
+import {sub} from 'frontend-js-web';
+import React, {Ref, useEffect, useMemo, useRef, useState} from 'react';
 
+import useId from '../hooks/useId';
 import {Locale, Translations} from './TranslationAdminContent';
+import TranslationAdminItem from './TranslationAdminItem';
 import TranslationAdminModal from './TranslationAdminModal';
 
+const DISPLAY_TYPE = {
+	default: 'default',
+	horizontal: 'horizontal',
+} as const;
+
+type DisplayType = 'default' | 'horizontal';
```

Comment:
A convention for global constants is all caps, so this should be `DEFAULT: 'default'` etc.

A type with a union of values is perfectly fine. An alternative solution that's handy sometimes is an enum, like we use [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/types.ts).

Reply:
I have resent it here with the uppercase values :D https://github.com/liferay-frontend/liferay-portal/pull/3913